components/alibi-explain-server/alibiexplainer/explainer.py

```diff
@@ -45,11 +46,13 @@
 GRPC_MAX_MSG_LEN = 1000000000
 TENSORFLOW_PREDICTOR_URL_FORMAT = "http://{0}/v1/models/:predict"
 SELDON_PREDICTOR_URL_FORMAT = "http://{0}/api/v0.1/predictions"
+V2_PREDICTOR_URL_FORMAT = "http://{0}/v2/models/{1}/infer"
 
 class Protocol(Enum):
     tensorflow_http = "tensorflow.http"
     seldon_http = "seldon.http"
     seldon_grpc = "seldon.grpc"
+    v2_http = "kfserving.http"
```

Comment:
Just wondering for consistency whether we should call it kfserving everywhere - I guess later on we may do a full on refactor to just rename it to v2 everywhere